pkg/controller/install/install_controller.go

```diff
@@ -132,6 +134,12 @@ func (r *ReconcileInstall) transform(instance *servingv1alpha1.Install) error {
 
 // Apply the embedded resources
 func (r *ReconcileInstall) install(instance *servingv1alpha1.Install) error {
+	// Ensure needed prerequisites are installed
+	for _, f := range platformPrereqFuncs {
+		if err := f(r.client, r.scheme, instance.Spec.Namespace); err != nil {
```

Comment:
Spec.Namespace is likely "" here. Problem?

Reply:
Not a problem - I tested with it both set and unset. We need to pass some Namespace when checking if namespace-scoped resources, but an empty string works for that purposes just as well as a real namespace.